locksmith/src/worker/taskUtils/timeout.ts

```diff
@@ -0,0 +1,15 @@
+import { JobHelpers, Task } from 'graphile-worker'
+
+const rejectAfter = (duration: number) =>
+  new Promise((_, reject) => {
+    setTimeout(() => {
+      reject(new Error('Timeout'))
```

Comment:
Can we customize the error message so we have an easy way to identify jobs that are timing out please?